### PR TITLE
Features/1820 high quantity consumables

### PR DESCRIPTION
**Pull request by ScienceNFaith**
I love this project and greatly appreciate how clean the code is. It was very easy to get familiar with the code base and start contributing. That being said, I'm a bit new to large scale git projects, so some of my commits and merges may be weird. 

I have adjusted the consumables_users table and corresponding consumableAssignment model to include a quantity. The migration sets default =1 so existing assignments still function as intended. I included some basic validation and bounds checking in the consumablesController, and allow for checkout of negative quantities. This signifies consuming the consumable, and this will not increase the number remaining. 
I also changed the account view_assets page to give a sum of each type of consumable checked out to that user rather than a list by name of each time one was assigned. This number changes when consumables are consumed and represents how many of that consumable that user has on hand. This should never be negative.

Hopefully this is a step in the direction you were hoping to go, but is definitely useful in my use case and seems like it will be for others as well, as per issue #1820. Thanks for a great inventory solution!

### CODE REVIEW

**Comment by snipe** [comment]
Hi there - can you rebase an re-push? I'm seeing a lot of my own commits in this PR, just makes it hard to parse through the changes.

**Comment by ScienceNFaith** [comment]
Alright, should be better now. Cherry-picked all the right things and should be good to go. Boy git can get complicated if I do it wrong, and it has been a learning experience for sure. Thanks again for a great project.

**Comment by ScienceNFaith** [comment]
It looks like the api controllers are broken in the develop branch. I didn't touch those, but this pull request is failing checks on trying to use them.

**Comment by dmeltzer** [comment]
Hi,

Sometimes travis is weird with tests.  That being said, you should be able to install dev dependencies (composer install) and then from the root directory run `vendor/bin/codecept run api`

**Comment by ScienceNFaith** [comment]
Looks like there's some development going into the tests and api right now and it's using some files I don't have in my repo when I forked. I think I'll stop trying to get it past travis tests if it's a moving target because it would require me to pull from the develop branch, which would make this pull request really messy. Is there anything you guys would like me to do for this pr or would you rather I just leave it as is?

**Comment by Benki** [comment]
Quantity input  and checkout for a department in the standard code would be great.

**Comment by aadrina** [comment]
Hi guys, i tried this pull. everything went smooth until i try to checkout consumable, it turned to error page. pls help
